src/connectors/xiami.js

```diff
@@ -1,22 +1,19 @@
 'use strict';
 
-Connector.playerSelector = '#player-main';
+Connector.playerSelector = '.play-bar';
 
-Connector.trackSelector = '#J_trackName';
+Connector.trackSelector = '.music .info .title';
 
-Connector.albumSelector = '.ui-track-current .c3>a';
+Connector.artistSelector = '.music .info .singers';
 
-Connector.artistSelector = '#J_trackInfo>a:nth-child(2)';
-
-Connector.playButtonSelector = '.play-btn';
-
-Connector.currentTimeSelector = '#J_positionTime';
-
-Connector.trackArtSelector = '#J_playerCoverImg';
+Connector.trackArtSelector = '.music .cover-link .active img';
 
 Connector.getDuration = () => {
-	let text = Connector.getCurrentTime() ? $('#J_durationTime').text() : '';
-	return Util.stringToSeconds(text);
+	let text = $('.audio-progress .range .bar .handle').val();
+	let str = text.split('/');
+	return str[0];
 };
 
-Connector.getUniqueID = () => $('.ui-track-current .c1').data('id');
```

Comment:
Is there no way to get uniqId?

Reply:
I could not find a way to get an uniqId from the website.

Reply:
`$('.play-bar .content .title').attr('href').split('?')[0].split('/song/')[1]`

![image](https://user-images.githubusercontent.com/434428/50686820-d26cf300-1059-11e9-832e-2c95e2b110b2.png)

![image](https://user-images.githubusercontent.com/434428/50686853-e4e72c80-1059-11e9-9078-c58fde938f28.png)

Reply:
@KwonDae Can you apply the changes that @naoyeye suggested?

Reply:
@naoyeye 
Oh Thanks!!
I'll try it now. But can you explain the getUniqueId why it is needed?

Reply:
@KwonDae read the docblock above the following: https://github.com/web-scrobbler/web-scrobbler/blob/master/src/core/content/connector.js#L251